insights/parsers/tests/test_losetup.py

```diff
@@ -0,0 +1,45 @@
+from insights.parsers.losetup import LoSetup
+from insights.tests import context_wrap
+
+LOSETUP_RHEL8 = """
+NAME       SIZELIMIT OFFSET AUTOCLEAR RO BACK-FILE       DIO LOG-SEC
+/dev/loop2      4096    256         0  1 /root/disk2.img   0    1024
+/dev/loop0         0      0         1  0 /root/disk1.img   1     512
+"""
+
+LOSETUP_RHEL7 = """
+NAME       SIZELIMIT OFFSET AUTOCLEAR RO BACK-FILE
+/dev/loop3      4096    256         0  1 /root/disk3.img
+"""
+
+
+def test_losetup():
```

Comment:
Please add a doctest, the previous example I mentioned ntp_sources parser, you can reference it's test for doc test.

Reply:
I have added the doctest.